project/models.py

```diff
@@ -35,26 +35,32 @@ def to_json(self):
 
 
 class User:
+    """
+    class for users
+    """
     def __init__(self, *args):
```

Comment:
@3Nakajugo, instead of parsing args and then use index to get the values why not try this 
 ```suggestion
    def __init__(self, *kwargs):
           self.user_id = kwargs.get('user_id`)
```
Then add all the other fields using keywords instead of indexes.